src/TSHelper.ts

```diff
@@ -727,7 +730,13 @@ export function getDeclarationContextType(
         return ContextType.NonVoid;
     }
 
-    // Walk up to find @noSelf or @noSelfOnFile
+    // When using --noSelf and the signature being used is in a file targeted by the program apply the --noSelf rule.
```

Comment:
- Option name in this comment is outdated now
- `being used` -> `is defined`?
- `apply the --noSelf rule` part sounds confusing, did you mean `@noSelf rule`?